node/npm_dependencies.ts

```diff
@@ -185,6 +254,15 @@ export const vendorNPMSpecifiers = async ({
     target: 'es2020',
   })
 
+  for (const { barrelName, types } of ops) {
+    if (!types) continue
+    // we're updating the output instead of adding this to the input,
```

Comment:
It's really annoying that esbuild doesn't make it possible to preserve certain comments in the output (only if they follow the pattern of [legal comments](https://esbuild.github.io/api/#legal-comments)).

Otherwise it'd be super easy for us to add the comment to the barrel file and avoid getting in the business of re-writing files.

😞

Reply:
Yeah, also checked legal comments :/ The current way is pretty much the easiest, sadly ...